packages/esbuild-plugin/package.json

```diff
@@ -1,40 +1,71 @@
 {
     "name": "@datadog/esbuild-plugin",
     "packageManager": "yarn@4.0.2",
+    "version": "0.0.13-4",
     "license": "MIT",
     "author": "Datadog",
+    "description": "Datadog ESBuild Plugin.",
```

Comment:
Nit: you may want to drop the `.` at the end of all your descriptions

Reply:
Will do, but why?

Reply:
Almost no-one uses `.` at the end of descriptions